manager/notifier.go

```diff
@@ -164,6 +165,38 @@ func (n *notifier) sendPagerDutyNotification(serviceKey string, a *Alert) error
 	return nil
 }
 
+func (n *notifier) sendHipChatNotification(authToken string, roomId int32, color string, notify bool, a *Alert) error {
+	// https://www.hipchat.com/docs/apiv2/method/send_room_notification
+	incidentKey := a.Fingerprint()
+	buf, err := json.Marshal(map[string]interface{}{
+		"color":  color,
+		"message":  a.Description,
```

Comment:
The description tends to be pretty light, I think we'll want a bit more than that. I'd suggest a lighter version of what goes in the email, and a link (configurable via flag) back to the alertmanager